### ISSUE TITLE
Seamonkey Support

### ISSUE THREAD

**Issue by rn10950** [honestbleeps/Reddit-Enhancement-Suite#2180]
I would love to use RES on SeaMonkey, Is there an easy way to go about this?

**Comment by erikdesjardins** [comment]
You could try some of the things described in [these posts,](https://www.reddit.com/r/enhancement+resissues/search?q=seamonkey&restrict_sr=on&sort=relevance&t=all) but it appears that it stopped working entirely [in Seamonkey 2.29](https://www.reddit.com/r/Enhancement/comments/2kbdsc/does_anyone_know_how_to_get_res_to_work_on/cners1h), so I somewhat doubt that it will have started working again.

We can't really devote any time to getting RES to work on Seamonkey because the userbase is so small - but if you do figure something out, feel free to post it to /r/Enhancement for the benefit of any other users.

**Comment by rodneyrod** [comment]
Unfortunately this is an issue which has recently affected me (I've recently had to change over from FF) and as a result I'm unable to use RES.

Has there been any progress made on this?
I'm currently a periodic donor, and if this support was added I would definitely add more when I can.

**Comment by erikdesjardins** [comment]
Unfortunately it's even harder now for RES to work with SeaMonkey, because we depend on a few changes made in Firefox 48 (some JavaScript language features and Firefox API changes).
And it will only get worse when we switch to Firefox WebExtensions, which I do not believe SeaMonkey has any support for (yet).
The future of RES on SeaMonkey looks very bleak, sorry.

**Comment by rodneyrod** [comment]
Most of those changes are backported to Seamonkey anyway, so it's entirely possible to port the plugin once the appropriate features have filtered over.

**Comment by erikdesjardins** [comment]
Still, in all likelihood we won't have first-party support for SeaMonkey (again, the userbase is too small). If RES happens to start working whenever it catches up to Firefox, great; if not ¯\\\_(ツ)_/¯

**Comment by rodneyrod** [comment]
So there's no sort of BountySource where I can direct donations to fixing specific issues? Because like I said before, if this was to be fixed, I would be willing to contribute funding to get the ball rolling.

**Comment by honestbleeps** [comment]
the problem is that there are so many dependencies, and the number of hours required to get support would be quite high -- high enough that a bounty would probably have to be a considerable amount of money...

dependency wise, we'd need Seamonkey to support all of the new Firefox APIs/structures that RES needs. No bounty will change that.

Even once it gets to that point, if RES doesn't just automatically work again, someone would need to spend a considerable amount of time testing, debugging and fixing and it more than likely wouldn't be just a day or two of work.

Then, beyond that, there'd be continued maintenance (you're going to be upset if you pay a bounty, it works for a month, then Seamonkey makes a change and it breaks again, right?)... 

Basically, the amount of effort required to get it running on a browser that an extremely tiny number of people use is unlikely to be in line with what a bounty would be...

**Comment by antdude** [comment]
Do I assume there's still no support for the updated SeaMonkey  versions (e.g., 2.53.10.2)?
